packages/grant-explorer/src/features/round/ViewCartPage/RoundInCart.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { CartProject, PayoutToken } from "../../api/types";
+import { useRoundById } from "../../../context/RoundContext";
+import { ProjectInCart } from "./ProjectInCart";
+
+export function RoundInCart(
+  props: React.ComponentProps<"div"> & {
+    roundCart: CartProject[];
+    selectedPayoutToken: PayoutToken;
+    handleRemoveProjectFromCart: (projectsToRemove: string) => void;
+    payoutTokenPrice: number;
+  }
+) {
+  const round = useRoundById(
+    String(props.roundCart[0].chainId),
+    props.roundCart[0].roundId
+  ).round;
+  const minDonationThresholdAmount =
+    round?.roundMetadata?.quadraticFundingConfig?.minDonationThresholdAmount;
+  return (
+    <div className="my-4 bg-grey-50 rounded-xl">
+      <div className="flex flex-row pt-4 px-4">
+        <p className="text-xl  font-semibold">{round?.roundMetadata?.name}</p>
+        <p className="text-lg font-bold ml-2">({props.roundCart.length})</p>
+      </div>
+      {minDonationThresholdAmount && (
+        <div>
+          <p className="text-sm pt-2 pb-4 px-4">
+            Your donation to each project must be valued at{" "}
+            {minDonationThresholdAmount} USD or more to be eligible for
+            matching.
+          </p>
+        </div>
+      )}
+      {props.roundCart.map((project: CartProject, key: number) => (
```

Comment:
Is it necessary to specify types here?

Reply:
WE LOVE EXPLICIT TYPES, WE LOVE EXPLICIT TYPES.

just kidding, fixed.